Meta/Lagom/CMakeLists.txt

```diff
@@ -355,6 +355,21 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
     target_link_libraries(LibCoreMinimal PRIVATE rt)
 endif()
 
+# LibCore
```

Comment:
yeah this can't go here. we don't build LibCore when LAGOM_TOOLS_ONLY. See LibCore/CMakeLists.txt

All of this junk should be moved to that file anyway. Around line 108 or so.